src/website/app/demos/Table/examples/selectable.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+import Table from '../../../../../library/Table';
+import sharedData from '../shared/data';
+
+export default {
+  id: 'selectable',
+  title: 'Selectable Rows',
+  description: `Allow users to select rows with the \`selectable\` prop.
+Rows with a \`disabled\` property set to \`true\` will render a disabled
+checkbox. You can set the initially selected rows with the
```

Comment:
Instead of "You can set the initially selected rows with the \`defaultSelectedRows\` prop", maybe "Rows can be rendered selected on initial load with the \`defaultSelectedRows\` prop.